src/main/java/com/microsoft/azure/vmagent/AzureVMAgentTemplate.java

```diff
@@ -581,6 +588,22 @@ public ListBoxModel doFillLocationItems(
             return model;
         }
 
+        public ListBoxModel doFillStorageAccountTypeItems(
+                @QueryParameter final String virtualMachineSize)
+                throws IOException, ServletException {
+
+            ListBoxModel model = new ListBoxModel();
+
+            model.add(Constants.STORAGE_ACCOUNT_TYPE_STANDARD);
```

Comment:
A better way to call `add(displayName, value)` the current overload. So you can do something like, to display as `Standard` and set the vaule to `SkuName.STANDARD_LRS`. Later you can write less String comparisons. Might be less constants definition as well.